scripts/gen_openapi.sh

```diff
@@ -29,89 +29,55 @@ export VAULT_ADDR=http://127.0.0.1:8200
 echo "Mounting all builtin plugins..."
 
 # Enable auth plugins
-codeLinesStarted=false
-
-while read -r line; do
-    if [[ $line == *"credentialBackends:"* ]] ; then
-        codeLinesStarted=true
-    elif [[ $line == *"databasePlugins:"* ]] ; then
-        break
-    elif [ $codeLinesStarted = true ] && [[ $line == *"consts.Deprecated"* || $line == *"consts.PendingRemoval"* || $line == *"consts.Removed"* ]] ; then
-        auth_plugin_previous=""
-    elif [ $codeLinesStarted = true ] && [[ $line =~ ^\s*\"(.*)\"\:.*$ ]] ; then
-        auth_plugin_current=${BASH_REMATCH[1]}
-
-        if [[ -n "${auth_plugin_previous}" ]] ; then
-            echo "enabling auth plugin: ${auth_plugin_previous}"
-            vault auth enable "${auth_plugin_previous}"
-        fi
-
-        auth_plugin_previous="${auth_plugin_current}"
-    fi
-done <../../vault/helper/builtinplugins/registry.go
-
-if [[ -n "${auth_plugin_previous}" ]] ; then
-    echo "enabling auth plugin: ${auth_plugin_previous}"
-    vault auth enable "${auth_plugin_previous}"
-fi
+vault auth enable "alicloud"
+vault auth enable "approle"
+vault auth enable "aws"
+vault auth enable "azure"
+vault auth enable "centrify"
+vault auth enable "cert"
+vault auth enable "cf"
+vault auth enable "gcp"
+vault auth enable "github"
+vault auth enable "jwt"
+vault auth enable "kerberos"
+vault auth enable "kubernetes"
+vault auth enable "ldap"
+vault auth enable "oci"
+vault auth enable "oidc"
+vault auth enable "okta"
+vault auth enable "pcf"
+vault auth enable "radius"
+vault auth enable "userpass"
```

Comment:
[non-blocker]
am 100% for removing the `if` logic and making the script more declarative, but this seems like a great use case for a loop?
```suggestion
readonly AUTH_PLUGINS=(
  # AUTH_PLUGINS - START
  alicloud
  approle
  aws
  azure
  centrify
  cert
  cf
  gcp
  github
  jwt
  kerberos
  kubernetes
  ldap
  oci
  oidc
  okta
  pcf
  radius
  userpass
  # AUTH_PLUGINS - END
)

for auth_plugin in "${AUTH_PLUGINS[@]}"; do
  vault auth enable "${auth_plugin}"
done
```

Reply:
Unfortunately, this would break the test that verifies the consistency. It regex-scans the script for `vault auth enable <...>`.